src/lib/CookieDecoder.ts

```diff
@@ -4,7 +4,14 @@ import { WalletInfoEntry, WalletType } from './WalletInfo';
 import { Utf8Tools } from '@nimiq/utils';
 import { AccountInfoEntry } from './AccountInfo';
 import CookieJar from './CookieJar';
-import { LABEL_MAX_LENGTH } from '@/lib/Constants';
+import {
+    ACCOUNT_DEFAULT_LABEL_KEYGUARD,
+    ACCOUNT_DEFAULT_LABEL_LEDGER,
+    ADDRESS_DEFAULT_LABEL_KEYGUARD,
+    LABEL_MAX_LENGTH,
+    ADDRESS_DEFAULT_LABEL_LEDGER,
+    ACCOUNT_DEFAULT_LABEL_LEGACY,
```

Comment:
The order of the constants is a bit mixed. The accounts_* and address_* constants should be grouped together.